ops/storage.py

```diff
@@ -187,15 +186,15 @@ def drop_notice(self, event_path: str, observer_path: str, method_name: str) ->
             (event_path, observer_path, method_name),
         )
 
-    def notices(self, event_path: str | None = None) -> _NoticeGenerator:
+    def notices(self, event_path: str | None = None) -> _Notices:
```

Comment:
I think this type already exists, but `list[_Notice]` seems clearly to me. Can we change to that?